tests/test_mathml2sympy.py

```diff
@@ -118,12 +125,30 @@ def test_log(self):
         self.assert_equal('<apply><log/><ci>x</ci></apply>',
                           [sympy.log(sympy.Symbol('x'), 10)])
 
+    def test_log_with_base(self):
+        # numeric base
+        self.assert_equal('<apply><log/><logbase><cn>3</cn></logbase><ci>x</ci></apply>',
+                          [sympy.log(sympy.Symbol('x'), sympy.Float(3.0))])
+        # symbolic base
+        self.assert_equal('<apply><log/><logbase><ci>y</ci></logbase><ci>x</ci></apply>',
+                          [sympy.log(sympy.Symbol('x'), sympy.Symbol('y'))])
+
     def test_diff(self):
         time = sympy.Symbol('time')
         V = sympy.Function('V')
         self.assert_equal('<apply><diff/><bvar><ci>time</ci></bvar><ci>V</ci></apply>',
                           [sympy.Derivative(V(time), time)])
 
+    def test_diff_with_order(self):
+        time = sympy.Symbol('time')
+        V = sympy.Function('V')
+        self.assert_equal('<apply>'
+                          '<diff/>'
+                          '<bvar><ci>time</ci><degree><cn>2</cn></degree></bvar>'
+                          '<ci>V</ci>'
+                          '</apply>',
+                          [sympy.Derivative(V(time), time, 2)])
+
     def test_piecewise(self):
         x = sympy.Symbol('x')
         self.assert_equal('<piecewise>'
```

Comment:
Should we add a longer one with more options than one? Maybe conflicting ones too to check if they're implemented the same.

For example:
if x > 0: y = 10
if x > 1: y = 20
if x > 2: y = 30
else: y= 0

This should return 10 for x = 3, because that's the first matched option

Reply:
Looks like Sympy does that.

```
>>> import sympy
>>> x = sympy.Symbol('x')
>>> expr = sympy.Piecewise((10, x > 0), (20, x > 1), (30, x > 2), (0, True))
>>> expr.subs({x: 3})
10
```

Reply:
Good test to add for the model evaluation part.